app-text/zathura/zathura-0.5.2-r5.ebuild

```
@@ -40,7 +39,8 @@ RDEPEND="${DEPEND}"
 
 BDEPEND="
 	test? ( dev-libs/appstream-glib
```

I prefer this as it's less noisy on diffs:
```suggestion
	test? (
		dev-libs/appstream-glib
		...
	)
```